pkg/dbsyncers/policy_syncer.go

```diff
@@ -24,12 +25,13 @@ const (
 	complianceStatusTableName = "compliance"
 )
 
+var log = ctrl.Log.WithName("policies-db-syncer")
```

Comment:
linters don't like global variables. global variable violates GoLang best practices (unless it's err definions)
this usually gives linter errors and reduces the code quality when doing static analysis in code check tools.
e.g. - https://goreportcard.com/report/github.com/stolostron/hub-of-hubs-status-sync

in addition, this log variable is now available in all dbsyncers.
this may cause human errors when other developers work on this code and they might use the global log and we may get wrong logging, which I guess is not what you meant.

Reply:
I agree. putting log (and also k8sClient I think) makes perfect sense to go into the generic db syncer.
at the end, every db syncer will use log + client for sure. 

minor changes are required for this. let's finish this PR while keeping things this way and then I'll change code to work with log + k8sClient inside the generic db syncer.